module4/projects/capstone/expectations.md

```diff
@@ -0,0 +1,180 @@
+
+<section class="call-to-action">
+Pro-Tip: Updating the README is a great way to get familiar with the project and make a first contribution on the job, too! 
+</section>
+
+### Guide for Navigating a Legacy Codebase
+As you are getting familiar with the codebase, consider the following questions:
+
+* Where will you start in the codebase, and why did you decide to start there?
+* What is familiar to you about the codebase?
+* Are you making any assumptions? If so, what are they?
+* How are you identifying what files are relevant to your issue?
+* What is the data flow in the application?
+* What is unfamiliar? (different file structures, new packages or gems, etc.)
+
+Take notes on your findings and questions. This will help you as you begin to work on your project as well as be a great resource to come back to when you start your first job.
+
+## Working Like a Developer
+After the onboarding and setup period, you will be assigned your first feature. Some features will be small and others will be large enough that you will need to break them into multiple issues. You should ask clarifying questions with your project manager as needed. There will be daily opportunities to do this synchronously in the daily check in, otherwise you can use Slack to communicate with your project manager or ask to schedule a call. Your project manager will be the final decision maker on the scope of the feature, but you are encouraged to ask questions and provide input.
+
+As the developer, you are responsible for:
+* Planning the feature (see below)
+* Writing the code to implement the feature
+* Testing the feature to verify it works as expected
+* Writing or updating the documentation for the feature
+* Ensuring your code does not break existing functionality or tests
+  * If existing tests break or are inadequate, update them. Be sure to document those changes and why there were needed in the PR.
+* Ensuring CI passes before requesting a review
+* Verifying your deployed changes are functioning as expected in production
+
+### Planning the Feature Looks Like:
+* Clarifying requirements of user stories with the project manager
+* Breaking down the feature into smaller issues if needed AND adding those issues to the project board
+  * Remember, PRs should be focused on a single feature or bug fix. If you are working on a large feature, break it into smaller pieces and submit them as separate PRs. Each PR should have a single purpose and address a single issue.
+* Choosing the appropriate tools, patterns, and technologies to implement the feature
+  * This might include discussing tradeoffs with the project manager or other developers
+
+<section class="dropdown">
+### Breaking Down a Feature Example
+
+If the assigned feature is **Users can sign up for a new account**
+
+You might break it down into the following issues after clarifying requirements with the project manager:
```

Comment:
I'm stuck on this idea, and not sure how much it matters (would be curious how you and others feel!) but in my experience it's more conventional to think of these sub-issues as acceptance criteria for a single issue. So instead of having multiple stories to track this issue, I think I'd encourage students to do this planning *within* the single card. If it's breaking down this feature into multiple stories, maybe they'd have a "sign up" and then a "login" issue.  Or if they're working on just one side of the stack, that could be broken down into FE/BE cards. But I'd expect each of those cards to have error handling, routing, etc as separate ACs within the story, rather than each AC be an issue itself

Reply:
This "Breaking down the Feature" could be instead a Story Tasking exercise like what is done here: https://www.pluralsight.com/resources/blog/guides/break-down-agile-user-stories-into-tasks-and-estimate-level-of-effort